contrib/add_license_to_files.jl

```diff
@@ -59,6 +59,7 @@ const skipfiles = [
     "../src/support/utf8.c",
     "../test/perf/micro/randmtzig.c",
     "../src/support/crc32c.c",
+    "../examples/quine.jl", # has license text in code
```

Comment:
should have said so at https://github.com/JuliaLang/julia/pull/19715. if that gets reverted, then the comment here can be removed or changed